prime-router/settings/STLTs/MO/mo-phd.yml

```diff
@@ -0,0 +1,114 @@
+- name: "mo-phd"
+  description: "Missouri Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "MO"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "mo-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/MO/MO-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "MOELR^2.16.840.1.114222.4.3.3.36^ISO"
+        receivingApplicationOID: "2.16.840.1.114222.4.3.3.36"
+        receivingFacilityName: "MODHSS^2.16.840.1.114222.4.3.2.2.1.163^ISO"
+        receivingFacilityOID: "2.16.840.1.114222.4.3.2.2.1.163"
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB: null
+        reportingFacilityName: "CDC PRIME"
+        reportingFacilityId: "11D2030855"
+        reportingFacilityIdType: null
+        suppressQstForAoe: true
+        suppressHl7Fields: "MSH-19-3, PID-15-3, ORC-14-1, OBR-17-1"
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: true
+        truncateHl7Fields: "MSH-3-1, PID-3-3, PID-3-6, ORC-2-2, ORC-7-10-5, OBR-2-2,\
+        \ OBR-3-2, SPM-2-1-2, SPM-2-2-2, ORC-21-1, OBX-23-1, BX-15.1, OBX-23.10 "
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "STANDARD"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: true
+          truncateHl7Fields:
+            - "MSH-3-1"
+            - "PID-3-3"
+            - "PID-3-6"
+            - "ORC-2-2"
+            - "ORC-7-10-5"
+            - "OBR-2-2"
+            - "OBR-3-2"
+            - "SPM-2-1-2"
+            - "SPM-2-2-2"
+            - "ORC-21-1"
+            - "OBX-23-1"
+            - "BX-15.1"
+            - "OBX-23.10"
+          customLengthHl7Fields: {}
```

Comment:
These should be:
```
  - "OBX-15-1"
  - "OBX-23-10"
```
This is going to be tedious, but can you check these fields are getting truncated by adding an integration test. We've had issue with truncation before and some fields were not getting truncated depending on where they were on the message. Specially these fields:
```
- "SPM-2-1-2"
- "SPM-2-2-2"
- "ORC-7-10-5"
```
I don't know if the syntax for truncating them in the UP is the right one. We haven't had to truncate fields in the UP that are that nested.

Reply:
Truncate usually means shorten, so what does truncate mean in this case?

Reply:
Truncation does not work properly. I created two tickets for this: https://app.zenhub.com/workspaces/engagement-6166edbd409257001e09f1b2/issues/gh/cdcgov/prime-reportstream/15110 and https://app.zenhub.com/workspaces/engagement-6166edbd409257001e09f1b2/issues/gh/cdcgov/prime-reportstream/15109 
can we merge this PR and go back to address this when these are fixed? @victor-chaparro